app/views/layouts/application.html.erb

```diff
@@ -27,6 +27,15 @@
   </head>
 
   <body class="autolab">
+    <div id="fb-root"></div>
+    <script>(function(d, s, id) {
+      var js, fjs = d.getElementsByTagName(s)[0];
+      if (d.getElementById(id)) return;
+      js = d.createElement(s); js.id = id;
+      js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.3&appId=259908887444831";
+      fjs.parentNode.insertBefore(js, fjs);
+    }(document, 'script', 'facebook-jssdk'));</script>
```

Comment:
Looks like it adds this:

![screenshot 2015-04-25 21 52 31](https://cloud.githubusercontent.com/assets/5544532/7335532/6810657e-eb95-11e4-9362-b622b20f0a69.png)